sklearn/feature_extraction/_dict_vectorizer.py

```diff
@@ -35,6 +36,10 @@ class DictVectorizer(TransformerMixin, BaseEstimator):
     a feature "f" that can take on the values "ham" and "spam" will become two
     features in the output, one signifying "f=ham", the other "f=spam".
 
+    When feature values are iterables of strings (not mappings), this
+    transformer will iterate over the values to perform the same transformation
```

Comment:
"to perform the same transformation to every element" seems a bit vague.
How about:
"and will similarly encode each string value as a binary feature."